File: service-manager/src/main/java/com/mss/servicemanager/DTO/ServiceDetailsDto.java
package com.mss.servicemanager.DTO;

import lombok.Data;
import java.util.List;
import java.util.UUID;

@Data
public class ServiceDetailsDto {
    private UUID id;
    private String name;
    private String description;
    private String version;
    private String endpoint;
    private boolean status;
    private String configuration;
    private String pricing;
    private List<GroupDto> groups;
    private List<UserDto> users;
}

File: MssPayProSaasFront/src/app/modules/api-service/ServiceDetailsDto.ts
import { GroupDto } from "../group/group.dto";
import { UserDTO } from "../user/user.dto";

export interface ServiceDetailsDto {
  id: string;
  name: string;
  description: string;
  version: string;
  endpoint: string;
  status: boolean;
  configuration: string;
  pricing?: number; 
  groups: GroupDto[];
  users: UserDTO[];
}
